persistence/src/test/scala/hmda/persistence/processing/HmdaFileParserSpec.scala

```diff
@@ -0,0 +1,55 @@
+package hmda.persistence.processing
+
+import java.time.Instant
+
+import akka.testkit.TestProbe
+import com.typesafe.config.ConfigFactory
+import hmda.actor.test.ActorSpec
+import hmda.parser.fi.lar.LarCsvParser
+import hmda.persistence.CommonMessages.GetState
+import hmda.persistence.processing.HmdaRawFile._
+import hmda.persistence.processing.HmdaFileParser._
+
+class HmdaFileParserSpec extends ActorSpec {
+  import hmda.model.util.FITestData._
+
+  val config = ConfigFactory.load()
+
+  val submissionId = "12345-2017-1"
+
+  val hmdaFileParser = createHmdaFileParser(system, submissionId)
+
+  val probe = TestProbe()
+
+  val timestamp = Instant.now.toEpochMilli
+  val lines = fiCSV.split("\n")
+  val badLines = fiCSVParseError.split("\n")
+
+  override def beforeAll() {
+    super.beforeAll()
```

Comment:
What is this `override def beforeAll()` doing? I might be missing something, because it looks to me like it ends up being a no-op / empty method.